samples/csharp_dotnetcore/55.facebook-adapter/README.md

```diff
@@ -0,0 +1,106 @@
+﻿# EchoBot using Facebook Adapter
+
+Bot Framework v4 echo bot using facebook Adapter sample.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to create a simple echo bot that connects with Facebook to respond to messages.
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 2.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+## To try this sample
+
+- Clone the repository
+
+   ```bash
+    git clone https://github.com/Microsoft/BotBuilder-Samples.git
+   ```
+
+- In a terminal, navigate to `BotBuilder-Samples/samples/csharp_dotnetcore/55.facebook-adapter`
+
+- Connect the bot with Facebook by following the instructions below.
+
+- Run the bot from a terminal or from Visual Studio, choose option A or B.
+
+  A) From a terminal
+
+  ```bash
+  # run the bot
+  dotnet run
+  ```
+
+  B) Or from Visual Studio
+
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `BotBuilder-Samples/samples/csharp_dotnetcore/55.facebook-adapter` folder
+  - Select `FacebookAdapterBot.csproj` file
+  - Press <kbd>F5</kbd> to run the project
+
+
+### Connect the bot with Facebook
+
+- Create a Facebook Account for Developers (https://developers.facebook.com/).
```

Comment:
These steps should have numbers, you'll need them to indicate how to set the tokens in appsettings.json